zone/bot_raid.cpp

```diff
@@ -296,4 +298,27 @@ void Bot::ProcessBotGroupAdd(Group* group, Raid* raid, Client* client, bool new_
 	raid->GroupUpdate(raid_free_group_id);
 }
 
+void Client::SpawnRaidBotsOnConnect(Raid* raid) {
+	std::list<BotsAvailableList> bots_list;
+	database.botdb.LoadBotsList(CharacterID(), bots_list);
+	std::vector<RaidMember> r_members = raid->GetMembers();
+	for (const auto& m: r_members) {
+		if (strlen(m.member_name) != 0) {
+
+			for (const auto& b: bots_list) {
```

Comment:
Should we check if bots_list is empty? Not that it’ll change anything. But we can just skip this sort if it’s empty.

Reply:
Might as well, added that in.